aiohttp/compression_utils.py

```diff
@@ -79,8 +87,8 @@ async def compress(self, data: bytes) -> bytes:
                 self._max_sync_chunk_size is not None
                 and len(data) > self._max_sync_chunk_size
             ):
-                return await asyncio.get_event_loop().run_in_executor(
-                    self._executor, self.compress_sync, data
+                return await asyncio.get_running_loop().run_in_executor(
```

Comment:
Should we replace these low-level calls with asyncio.to_thread() now that we don't support 3.8?